torch/utils/data/dataloader.py

```diff
@@ -313,6 +313,8 @@ def __len__(self):
             # `DataLoader`, save the returned value in `self._len_called`, and warn
             # if the iterator ends up yielding more than this number of samples.
             length = self._IterableDataset_len_called = len(self.dataset)
+            if self.batch_size is not None:
+                length = length // self.batch_size
```

Comment:
Almost correct! Need to test reminder in case of `not drop_last`.

Reply:
Not sure if this is what you mean but added a ```not self.drop_last``` condition in the next update. Let me know if this isn't what you meant!

Reply:
Oh I meant that if `drop_last is False`, then it is `ceil(length / batch_size)` since the last incomplete batch will be yielded. Here you are always computing `floor(length / batch_size)`

Reply:
Ohh okay will make that update right now.

Reply:
Oh my bad... I noticed that I had a typo: `remainder` lol